common/static/common/js/jasmine_stack_trace.js

```diff
@@ -0,0 +1,24 @@
+(function () {
+    /* globals jasmineRequire */
+    'use strict';
+
+    var OldExceptionFormatter = jasmineRequire.ExceptionFormatter(),
+        oldExceptionFormatter = new OldExceptionFormatter();
+
+    jasmineRequire.ExceptionFormatter = function () {
+        function ExceptionFormatter() {
+            this.message = oldExceptionFormatter.message;
+            this.stack = function (error) {
+                var errorMsg = null;
+
+                if (error) {
+                    errorMsg = error.stack.split('\n').slice(0, 10).join('\n');
+                }
+
+                return errorMsg;
+            };
+        }
+
+        return ExceptionFormatter;
+    };
+}());
```

Comment:
Here we are overriding `ExceptionFormatter ` of jasmine before it's initialization in karma-jasmine's boot.js `jasmineRequire.core(jasmineRequire)`. It's important because once the method has been initialized we can't override the `ExceptionFormatter `.